PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -914,20 +914,8 @@ def getMETUncertainties(self, process, metType, metModName, electronCollection,
         #---------------------------------------------------------------------
         # PFPhotons :
         #------------
-        if self._parameters["Puppi"].value or not self._parameters["onMiniAOD"].value:
-            cutforpfNoPileUp = cms.string("")
-        else:
-            cutforpfNoPileUp = cms.string("fromPV > 1")
-
-        pfNoPileUp = cms.EDFilter("CandPtrSelector",
-                                  src = pfCandCollection,
-                                  cut = cutforpfNoPileUp
-                                  )
-        addToProcessAndTask("pfNoPileUp"+postfix, pfNoPileUp, process, task)
-        metUncSequence += getattr(process, "pfNoPileUp"+postfix)
-
         pfPhotons = cms.EDFilter("CandPtrSelector",
-                                 src = cms.InputTag("pfNoPileUp"+postfix),
+                                 src = pfCandCollection,
```

Comment:
This change the meaning of the pfPhotons ?

Reply:
reverting to the old logic, just to be sure